### PR TITLE
Update escrow hijack hints

### PR DESCRIPTION
**Pull request by harryscholes**
I'm a CosmWasm n00b, but the following changes worked for me. Please review carefully :)

### CODE REVIEW

**Comment by orkunkl** [comment]
You are awesome!

File: docs/learn/hijack-escrow/edit-escrow-hints.md [hunk 1]
```diff
@@ -78,47 +69,55 @@ fn try_steal(
...
     let msg = ExecuteMsg::Steal {
-        destination: HumanAddr::from("bankvault"),
+        destination: "anybody".into(),
     };
-    let env = mock_env_height("anybody", &[], 900, 0);
-    let execute_res = execute(&mut deps, env, msg.clone());
-    assert!(execute_res.is_err());
+    let mut env = mock_env();
+    env.block.height = 900;
+    env.block.time = Timestamp::from_seconds(0);
```
Comment (orkunkl) [inline docs/learn/hijack-escrow/edit-escrow-hints.md]: You don't have to define both. env height should be enough.

**Review by orkunkl — CHANGES_REQUESTED** [review]
LGTM only one comment to change.

**Comment by harryscholes** [comment]
Cool, thanks!

**Review by orkunkl — APPROVED** [review]

**Pull request merged**